### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mserykh**

### CODE REVIEW

**Review by mserykh — COMMENTED** [review]
Looks good

File: index.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Welcome to my blog
 
-I'm glad you are here. I plan to talk about ...
+I am Marie Serykh and am aiming to become a Sofrwear Developer. To be precise a Front-End Developer.
```
Comment (dev-experience) [inline index.md]: Please fix the typo

**Review by dev-experience — CHANGES_REQUESTED** [review]
Please check my comment.

File: index.md (hunk 1 above)
Comment (mserykh) [reply index.md]: Done

**Review by dev-experience — APPROVED** [review]

**Pull request merged**